crates/burn-tensor/src/tests/ops/matmul.rs

```diff
@@ -31,7 +55,20 @@ mod tests {
     }
 
     #[test]
-    fn test_matmul_broadcast_1() {
+    fn test_int_matmul_broadcast_1() {
+        let device = Default::default();
+        let tensor_1 = TestTensorInt::<3>::from_ints([[[1, 7], [2, 3]]], &device);
+        let tensor_2 = TestTensorInt::from_ints([[[4, 7], [2, 3]], [[2, 5], [6, 3]]], &device);
+
+        let tensor_3 = tensor_1.matmul(tensor_2);
+        println!("{:?}", tensor_3);
+        let expected = TensorData::from([[[18, 28], [14, 23]], [[44, 26], [22, 19]]]);
+
+        tensor_3.into_data().assert_eq(&expected, false);
```

Comment:
Just be aware of this issue (false positive assertion): #1982

It may happen with TestTensorInt is i64 and you test data is i32 and vice versa.